packages/utils/src/isObject.ts

```diff
@@ -8,5 +8,8 @@ export default function isObject(thing: any) {
   if (typeof File !== "undefined" && thing instanceof File) {
     return false;
   }
+  if (typeof Date !== "undefined" && thing instanceof Date) {
+    return false;
+  }
```

Comment:
You will need to add a test for this use case

Reply:
I'm sorry to ask, but would you be able to give me a pointer as to how? I'm not familiar with the test configuration used on this project. 🙏

Reply:
@BenBeattieHood so it looks like we never added unit tests for `isObject()` whoops. Go ahead an add a new file `packages/utils/test/isObject.ts` with the following code:

```
import { isObject } from '../src';

const NON_OBJECTS = [
  'string',
  10,
  NaN,
  true,
  null,
  undefined,
];

const OBJECTS = [
  { plain: 'object' },
  new Object(),
  new Map(),
];

describe('isObject()', () => {
  it('returns false when a non-object is provided', () => {
    NON_OBJECTS.forEach((nonObject: string | number | boolean | null | undefined) => {
      expect(isObject(nonObject)).toBe(false);
    });
  });
  it('returns false when a File is provided', () => {
    const file = new File(["test"], "test.txt");
    expect(isObject(file)).toBe(false);
  });
  it('returns false when an array is provided', () => {
    expect(isObject(['foo'])).toBe(false);
  });
  it('returns true when an object is provided', () => {
    OBJECTS.forEach((object: any) => {
      expect(isObject(object)).toBe(true);
    });
  });
});

```

This will test all of the current code. I leave testing for your new code to you, should be fairly straight-forward